jupyter-docker/jupyter_localize_extension.py

```diff
@@ -23,13 +23,17 @@ def sanitize(self, pathstr):
   @gen.coroutine
   def localize(self, pathdict):
     """Treats the given dict as a string/string map and sends it to gsutil."""
+    all_success = True
     #This gets dropped inside the user's notebook working directory
     with open("localization.log", 'a', buffering=1) as locout:
       for key in pathdict:
         #NOTE: keys are destinations, values are sources
-        cmd = " ".join(["gsutil -m -q cp -R -c -e", self.sanitize(pathdict[key]), self.sanitize(key)])
-        locout.write(cmd + '\n')
-        subprocess.call(cmd, stderr=locout, shell=True)
+        cmd = ['gsutil', '-m', '-q', 'cp', '-R', '-c', '-e', self.sanitize(pathdict[key]), self.sanitize(key)]
+        locout.write(' '.join(cmd) + '\n')
+        code = subprocess.call(cmd, stderr=locout)
```

Comment:
@dionloy hopefully this is sufficient for https://github.com/DataBiosphere/leonardo/issues/293 ?

Reply:
LGTM for #293, thanks!